src/Popout.tsx

```diff
@@ -42,9 +41,25 @@ export class Popout extends React.Component<PopoutProps, {}> {
                 } else {
                     window.opener.${globalContext.id}.onChildClose.call(window.opener, '${id}');
                 }
+            }`;
+
+            // Use onload for most URL scenarios to allow time for the page to laod first
+            // Safari 11.1 is aggressive, so it will call onbeforeunload prior to the page being created.
+            unloadScriptContainer.innerHTML =
+                `
+            window.onload = function(e) {
+                ` +
+                onBeforeUnloadLogic +
```

Comment:
do you mean `${onBeforeUnloadLogic}` here?

Reply:
Could do that instead of the pluses, good point.